### PR TITLE
Add all features 

### PR DESCRIPTION
**Pull request by NatiDeme**
# Changes in this PR: 
- No linter errors.
- Used git flow.
- Implemented a list of categories
- Categories populate according to a specific filter
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- test included
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- Every page has a unique route within the SPA

### check video description:
[Video](https://www.loom.com/share/9cc7c0306a544a64a69cb03f6c3fef61)

### CODE REVIEW

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @NatiDeme , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Please kindly highlight a piece of code that you are proud of in your presentation video
- [x] Please kindly also ensure that there is only one `Detail` page instead of having `Agility`, `Strength`, and `Intelligence` as separate components. This will enable code reusability as most of the logic in those three components are the same.
- [x] Please kindly include unit tests for the components of the app and pure functions in Redux code.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/tests/Contain.test.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+// import renderer from 'react-test-renderer';
```
Comment (youmari) [inline src/tests/Contain.test.js]: - [x] please try to remove the unused commented code in order to keep your file clean and pro

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @NatiDeme , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

✅ No linter error
✅ PRO PR
## Suggested changes

- [x] Please as the previous reviewer state you should mention in the presentation video a piece of code that you are proud of 

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@NatiDeme  ✋🏼 
#### Your project is excellent. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You added a presentation video that includes a demo of the page and some codes that you are proud of.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by NatiDeme** [comment]
@ahmetbozaci Thank you very much!!! 
I really appreciate it and it gives me great joy to see support on your review!!

**Pull request merged**